session.go

```diff
@@ -156,6 +156,10 @@ func (s *Session) mux(remoteBegin *performBegin) {
 		txTransfer := s.txTransfer
 		// disable txTransfer if flow control windows have been exceeded
 		if remoteIncomingWindow == 0 || s.outgoingWindow == 0 {
+			debug(1, "TX(Session): Disabling txTransfer - window exceeded. remoteIncomingWindow:",
```

Comment:
was suspecting this to occur and added a debut trace, I think it's a wise one to keep anyway :)